### PR TITLE
Changed location of video module on landing page

### LINKED ISSUE #724: Location of Movie on Landing Page
With the choropleth on landing page, I wonder if it makes sense to move the movie somewhere? I think part of it is that it just looks a bit smashed up against the bottom of the choropleth map. 

A few ways we could deal with this:

1. Put a whitespace (or some other colorspace) buffer between the map and movie

2. Similar to 1., change the movie presentation a bit so that the module is consistent with our modular sections and have a title and subtitle for the movie before the movie itself

3. Move the movie down to after "What people are saying" and maybe change it to "Find out more..." or something.

Thoughts?

### PR DESCRIPTION
**Pull request by misaugstad**
The "What is Project Sidewalk?" video module on the landing page is now the 3rd module instead of the 4th.

Also changed choropleth module's background to white.

Resolves #724

### CODE REVIEW

File: public/css/main.css [hunk 1]
```diff
@@ -221,7 +221,6 @@ footer {
     color: #fff;
     font-family: raleway, sans-serif;
     font-size: 45px;
-    background: rgba(0,0,0,0.2);
```
Comment (manaswis) [inline public/css/main.css]: Why was this removed? This was part #699 fix.

Comment (misaugstad) [inline public/css/main.css]: No idea! Changed it back.
